src/vit-testing/snapshot-trigger-service/src/client/mod.rs

```diff
@@ -1,190 +1,2 @@
 pub mod args;
 pub mod rest;
-
```

Comment:
Has this file been deleted on porpuse?

Reply:
Yes , it was moved to integration tests since operations here working on top of snapshot service client. It caused issues when some library has dependency to snapshot service and voting tools and mainnet-tools. Code below is used integration tests only and break SPOR rule so i decided to move them to integration-tets